src/constants.ts

```diff
@@ -86,3 +86,10 @@ export const MARKET_DEPTH_END_PRICE_SLIPPAGE_PERC = 20;
 
 // Logging
 export const NUMBER_SOURCES_PER_LOG_LINE = 12;
+
+// RFQ Quote Validator expiration threshold
+
+// NOTE: Until worker component is created, we set an infinite expiry
+// rate which will allow us to start writing out records to the cache
+// table without causing all quotes to be invalid.
+export const RFQ_FIRM_QUOTE_CACHE_EXPIRY = Number.POSITIVE_INFINITY;
```

Comment:
Is this safe?

Reply:
It is, because of [this](https://github.com/0xProject/0x-api/pull/432/files#diff-fa95b01194bc9cd84720be5a299890448459cfb7c4b7faa452681c9f11f65907R42) safety pin